Documentation/compatibility/workflow-designer-accessibility.md

```diff
@@ -0,0 +1,123 @@
+## Accessibility improvements in WF workflow designer
+
+### Scope
+Minor
+
+### Version Introduced
+4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+The Windows Workflow Foundation (WF) workflow designer is improving how it works with accessibility technologies. These improvements include the following changes:
+- Tab order changed to left->right & top->bottom in some controls
+  - Initialize correlation window for setting correlation data for the InitializeCorrelation activity
+  - Content definition window for Receive, Send, SendReply, and ReceiveReply activities
+- More functions available via keyboard
+  - When editing the properties of an activity, fixed an issue where property groups could not be collapsed by keyboard the first time they are focused
```

Comment:
fixed an issue where property groups could not be collapsed --> property groups can be collapsed